libavcodec/pthread_slice.c

```diff
@@ -237,6 +237,8 @@ int ff_alloc_entries(AVCodecContext *avctx, int count)
 
 void ff_reset_entries(AVCodecContext *avctx)
 {
```

Comment:
this is in preparation for integration with our code base - we're supporting a mode in which we're in FRAME mode but do parallel slice decoding anyway in our own threads